src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -62,9 +64,20 @@ const CallTable = ({ namespace, name, submissionId, workflowId, callName, callOb
             size: { basis: 200, grow: 2 },
             headerRenderer: () => 'Call Caching Result',
             cellRenderer: ({ rowIndex }) => {
-              const { callCaching: { effectiveCallCachingMode, result } = {} } = callObjects[rowIndex]
+              const { shardIndex: index, attempt, callCaching: { effectiveCallCachingMode, result } = {} } = callObjects[rowIndex]
               if (effectiveCallCachingMode === 'ReadAndWriteCache' || effectiveCallCachingMode === 'ReadCache') {
-                return result ? h(TooltipCell, [result]) : div({ style: { color: colors.dark(0.7) } }, ['No Information'])
+                return result ? [
+                  h(TooltipCell, [result]),
+                  result === 'Cache Miss' && h(Link, {
+                    key: 'cc',
+                    style: { marginLeft: '0.5rem' },
+                    tooltip: 'Call Cache Debug Wizard',
```

Comment:
For accessibility, this should have an `aria-label` with this text as well.

Reply:
Would like to check that I did this right